docs/wiki/guides/current/Magnetometer.md

```diff
@@ -288,61 +302,112 @@ For the most accurate calibration results:
+
+If you do, don't touch the frame, and wait for the 15s timeout. Your old cal values will be retained.
+
+If you initiate a Mag Cal, and move the frame more than 350 deg/s, but fail to rotate the quad properly over the next 30s, your old cal values will be replaced by 'junk' Cal values, and your Mag data will be useless!
+
+:::
+
+### Calibration Technique
 
 A good way to cover every possible angle is to:
 
 - hold the quad by the battery
-- swing your arm around in a big circle, eg forwards -> up ->over -> backwards -> downwards, and keep swinging it around like this
+- swing your arm around in a big circle, eg forwards -> up -> over -> backwards -> downwards, and keep swinging it around like this
 - yaw the quad, randomly, left and right, at the same time
 - slowly rotate your whole body about its vertical axis by taking small steps, so that in 30s you have completed a full 360 degree rotation.
 
 If you know the local field direction, you may start and finish facing North, though this is not necessary.
 
 Smoothly completing full rotations about every 1.5-2 seconds works best, but keep in mind that you only have 30s.
 
-Check the `mag_calibration` CLI numbers after each run to see how consistent the values are. If each value is within 50 units of the last couple of runs, that's pretty good. Also check in the Sensors Tab that the Min and Max for each axis are approximately the same number.
+Another alternative method:
 
-:::warning
-Take great care not to initiate a Mag Cal accidentally! If you 'tap' the frame, but fail to rotate the quad properly after a Cal starts, your old cal values will be lost, and the Mag data will be useless!
-:::
+- hold the quad top down, nose back, with your arm by your side, and swing the arm 180° upwards so that the top of the quad points straight up into the sky
+- Yaw the quad 90°, and bring the arm back down all the way.
+- turn your whole body about 30°, and repeat the up, yaw, and drop movement.
+- Keep going until you've done this a whole bunch of times, and you've your whole body completely around, in the 30s period.
+
+Check the `mag_calibration` CLI numbers after each run to confirm that you are getting consistent values.
+
+You can be happy that things are pretty good if the values are within 20-30 field strengt units from run to run.
+
+The calibration can be confirmed by:
+
+- checking in the Sensors Tab, with the `Mag_Calib` debug, that the Min and Max for each axis are approximately the same number
+- checking in the Sensors Tab, with `Mag_Calib` debug, that the Normalised 'length of MagADC' value is reasonably constant at all angles.
+- comparing the reportd Heading of the quad (GPS tab or Setup page of Configurator) to the compass reading on your phone
+- pointing the Quad to True North (based on the phone) and checking that the arrow icon in the Map in the GPS Tab points straight upwards (the Map is always North at the top).
+- checking the Map in the GPS tab; the Heading I
```

Comment:
???

```suggestion
- checking the Map in the GPS tab; the Heading Icon
```

Reply:
thanks, I accidentally left that whole line there, removed it now.